aws-sns-lambda/test/module_test.go

```diff
@@ -0,0 +1,30 @@
+package test
```

Comment:
I followed the logic in https://github.com/chanzuckerberg/cztack/tree/main/bless-ca/test to write the test.